Project.toml

```diff
@@ -22,7 +22,7 @@ DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 
 [extensions]
-AlgebraicDynamicsDiffEqExt = ["DifferentialEquations", "OrdinaryDiffEq", "DelayDiffEq"]
+AlgebraicDynamicsDiffEqExt = ["OrdinaryDiffEq", "DelayDiffEq"]
```

Comment:
This should be broken up into 2 extensions one for Ordinary and one for Delay, since those features are orthogonal.

Reply:
done in most recent commit